### PR TITLE
Adding tests for "Unicode and passing strings" part 7

### PR DESCRIPTION
**Pull request by gabrieldemarmiesse**

### CODE REVIEW

File: docs/src/tutorial/strings.rst [hunk 1]
```diff
@@ -322,23 +315,17 @@ you can use the following approach.
...
+.. literalinclude:: ../../examples/tutorial/string/const_left_out.pyx
 
 However, in most other situations, such as for return values and
 variables that use specifically typedef-ed API types, it does matter
 and the C compiler will emit at least a warning if used incorrectly.
 To help with this, you can use the type definitions in the
-``libc.string`` module, e.g.::
-
-    from libc.string cimport const_char, const_uchar
+``libc.string`` module, e.g.:
```
Comment (scoder) [inline docs/src/tutorial/strings.rst]: Now that Cython has direct support for the ``const`` modifier, this ugly work-around should be removed from the docs and it should simply show how to declare and use APIs that use ``const``.
